### DISCUSSION TITLE
Netbox docker deploy on Synology with Portainer

### DISCUSSION THREAD

**Discussion by huntson** [netbox-community/netbox-docker#940] [Getting Help]
I've been trying for days now to get the Netbox docker deploy via Portainer on my Synology NAS.  When attempting to build the stack, the system can never find the env file and that's largely because I have no clue where the environment file needs to live.  I've tried putting the .env file on my NAS and hardcoding the location in all of the references to the file however it never works.  Any ideas there?

**Comment by AllenMedivh** [comment]
I totally given up on deploying Netbox through the Stack on Portainer and I was actually working on a Netbox that had the Limited Control on the stack list until someone on a forum pasted this from the Portainer Blog:

https://www.portainer.io/blog/using-env-files-in-stacks-with-portainer

I'm currently on my 3rd attempt to install Netbox this way but it's mostly mistakes I'm making when I try to setup the redis or postgres dependencies of Netbox not actually errors related to the ENVs not being found

Also you might want to follow this article as a base to install netbox:

https://spin.atomicobject.com/2021/11/08/netbox-linuxserver-ios-docker-image/

**Comment by mkevenaar** [comment]
I recently have written a blog post about this: https://kevenaar.name/netbox-how-to-run-it-on-your-synology/ @huntson
